### PR TITLE
Fixes #2052: Update outdated installation docs  

### LINKED ISSUE #2052: Update outdated installation docs
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Description :**
The installation docs for both front-end and back-end are quite outdated and need updation

**Current behaviour:**
Out dated docs

**Expected behaviour:**
Up-to date docs

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by abishekvashok**
Fixes #2052 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
Updates outdated installation docs making it easier for newcomers and existing developers to install the project in a convienient manner

### CODE REVIEW

File: api/docs/installation/localvir.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 The instructions on this page will guide you in setting up a local development
 environment in your system. First things first, BadgeYay needs 'Python 3' to run.
 Most of the distros come bundled with that but if it is not there please install it first.
+We will be installing postgress, but be sure you have `libxml2`, `zlib`, and `jpeg`. (Most distros do have them)
```
Comment (rajvaibhavdubey) [inline api/docs/installation/localvir.md]: Correct postgress to postgres

**Comment by abishekvashok** [comment]
@rajvaibhavdubey done

**Comment by abishekvashok** [comment]
@mariobehling pls review

**Comment by rajvaibhavdubey** [comment]
Thanks :+1:

**Pull request merged**